tutorials-apps/bulk-question-delete-app/src/components/main.js

```diff
@@ -0,0 +1,244 @@
+import React, { useState, useEffect } from 'react'
+import icon from "../assets/icon.png"
+import {
+  Box, Heading, Text, Button, Flex, Input, Checkbox, Avatar, toast, AlertDialog,
+  AlertDialogContent,
+  AlertDialogTitle,
+  AlertDialogDescription,
+  AlertDialogActions,
+  AlertDialogCancel,
+  AlertDialogAction,
+} from "@sparrowengg/twigs-react";
+import { SearchIcon } from '@sparrowengg/twigs-react-icons';
+import Question from './Question';
+import Logo from "../assets/appnest_logo.png"
+import axios from 'axios';
+import Loader from './loader';
+const Main = (props) => {
+  const [questions, setQuestions] = useState([])
+  const [surveyid, setSurveyId] = useState(0)
+  const [search, setSearch] = useState('')
+  const [loader, setLoader] = useState(false)
+  const [qids, setQids] = useState([])
+  const [btnLoader, setBtnLoader] = useState(false)
+  const [selectAll, setSelectAll] = useState(false)
+  const [showModal, setShowModal] = useState(false)
+  const [filteredQuestions, setFilteredQuestion] = useState([])
+  const getQuestions = async () => {
+    try {
+      setLoader(true);
+      let allQuestions = [];
+      let i = 1;
+      let response;
+
+      do {
+        response = await axios.get(`https://api.surveysparrow.com/v3/questions?page=${i}&survey_id=${surveyid}`, {
+          headers: {
+            Authorization: 'Bearer prTp291Zd5mN5eots8Bio2_eTaUWuvLOGquBfJ3jiAQELSxXnqfVN0H47HFcG5Cpzhgeic-QXknDhIKNpfL_uUGQ'
```

Comment:
@janarthananS1729 why are we hardcoding the access token we can get it from iparams right?, since env won't work?